### PR TITLE
Tweak and split plugin explanation

### PR DESCRIPTION
**Pull request by kenhys**
It will improve readability.

### CODE REVIEW

File: docs/output.md [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+For `<buffer>`, you can use any record field in `path` / `s3_object_key_format`.
+
+    path logs/${tag}/${foo}
+    <buffer tag,foo>
+      # parameters...
+    </buffer>
+
+See official article for more detail: [Config: Buffer Section - Fluentd](https://docs.fluentd.org/configuration/buffer-section)
+
+Note that this configuration doesn't work with fluentd v0.12.
```
Comment (ashie) [inline docs/output.md:39]: Adding a link to docs/v0.12.md will be more kindly.

**Comment by kenhys** [comment]
I'll send another PR for changelog.

File: docs/output.md [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+        path /var/log/fluent/s3
+        timekey 3600 # 1 hour partition
+        timekey_wait 10m
+        timekey_use_utc true # use utc
+      </buffer>
+      <format>
+        @type json
+      </format>
+    </match>
+
```
Comment (ashie) [inline docs/output.md:29]: Add a link to docs/credentials.md for `aws_key_id` and so on.

File: docs/input.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      aws_sec_key YOUR_AWS_SECRET_KEY
+      s3_bucket YOUR_S3_BUCKET_NAME
+      s3_region ap-northeast-1
+      add_object_metadata true
+
+      <sqs>
+        queue_name YOUR_SQS_QUEUE_NAME
+      </sqs>
+    </source>
+
```
Comment (ashie) [inline docs/input.md:24]: Add a link to docs/credentials.md for `aws_key_id` and so on.

**Review by ashie — CHANGES_REQUESTED** [review]

File: docs/output.md [hunk 4]
```diff
@@ -0,0 +1,445 @@
...
+For `<buffer>`, you can use any record field in `path` / `s3_object_key_format`.
+
+    path logs/${tag}/${foo}
+    <buffer tag,foo>
+      # parameters...
+    </buffer>
+
+See official article for more detail: [Config: Buffer Section - Fluentd](https://docs.fluentd.org/configuration/buffer-section)
+
```
Comment (cosmo0920) [inline docs/output.md:38]: Some users are struggled to understand how to work built-in placeholders feature.
Should we add a link which depicts built-in placeholders:
https://docs.fluentd.org/configuration/buffer-section#placeholders

And, should we add built-in placeholders usable paratemeters list?
Or, placeholders acceptable parameters should be described in each parameter descriptions.

File: docs/output.md [hunk 5]
```diff
@@ -0,0 +1,446 @@
...
+For [`<buffer>`](https://docs.fluentd.org/configuration/buffer-section), you can use any record field in `path` / `s3_object_key_format`.
+
+    path logs/${tag}/${foo}
+    <buffer tag,foo>
+      # parameters...
+    </buffer>
+
+See official article for available parameters and usage of placeholder in detail: [Config: Buffer Section](https://docs.fluentd.org/configuration/buffer-section#placeholders)
+
+Note that this configuration doesn't work with fluentd v0.12. See [v0.12](docs/v0.12.md) for v0.12 style.
```
Comment (ashie) [inline docs/output.md:40]: `docs/` isn't needed from here.

Comment (kenhys) [reply docs/output.md]: Fixed.

File: docs/output.md (hunk 4 above)
Comment (kenhys) [reply docs/output.md:39]: I'm not sure which is the best, anyway link is fixed to point placeholder.

**Review by ashie — APPROVED** [review]

**Pull request merged**